tests/Sniffs/Comparisons/Fixtures/EqualOperatorSniff/correct/Class.php

```diff
@@ -22,5 +22,10 @@ function test($a, $b): string
         if ($a === $b) {
             return 'Success';
         }
+
+        /** @phpcsSuppress BestIt.Comparisons.EqualOperator.EqualOperatorFound */
```

Comment:
Shouldn't this suppress the NotEqualOperatorFound? 
Or does this mean it throws an EqualOperatorFound Error/Warning when the NotEqualOperator is used?

Reply:
This is unit test for the supress helper, to check that the code sniffer does not throw the EqualOperatorFound error on !=.